### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bilzinho**
Clarified language regarding individual vs. corporate contribution, minor grammar and diction edits, added reference link in last paragragh.  Cf. Issue #455

### CODE REVIEW

**Review by bdirenzo-keysight — APPROVED** [review]

**Comment by bdirenzo-keysight** [comment]
@AsgerIversen FYI this is what you were looking to get reviewed

**Review by AsgerIversen — CHANGES_REQUESTED** [review]
Thanks @bilzinho. Looks good. Just one small comment..

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,35 +1,27 @@
 ## Contribute to OpenTAP
 
-Thank you for your interest in contributing to OpenTAP. Contributions include feature enhancement requests, issue reporting, bug fixes, etc.
+Thank you for your interest in contributing to the OpenTAP project. Contributions include feature enhancement requests, issue reporting, bug fixes, etc. as well as plugins and other new code.
```
[the remark this answers is not in the document]
Comment (bilzinho) [reply CONTRIBUTING.md:3]: Asger - I completely agree and have created a new version / update to the file that removed the language you note and adds a sentence on the fact that plugins are not governed by the CLA.

**Comment by bdirenzo-keysight** [comment]
@AsgerIversen I noticed you were merging doc stuff. This should also be good to go.

**Review by AsgerIversen — APPROVED** [review]

**Pull request merged**